### ISSUE TITLE
reproducibility issues

### ISSUE THREAD

**Issue by VikasKaja** [ukaukaaaa/GazeGNN#8]
hey I was trying to use your code of Gazegnn for reproducibility and dataset you have uploaded in github and I have never achieved accuracy above 80% my maximum accuracy on test data on the given dataset is 77% beyond that the accuracy never increased and also AUC scores of pneumonia class is very less for me.
Did you use any additional training data to train the model?
which pvig model have given you 83% accuracy on the dataset is it the pvig_ti_224_gelu or the larger ones because larger one always overfits to the data and smaller ones never able to predict pneumonia class.
